### PR TITLE
Update community codeowners

### PR DESCRIPTION
**Pull request by haydentherapper**
This adds both org admins as maintainers of the community repo to help with reviews.

### CODE REVIEW

**Review by cpanato — COMMENTED** [review]
I would say add myself as well :)

File: github-sync/github-data/sigstore/repositories.yaml
Comment (spencerschrock) [inline github-sync/github-data/sigstore/repositories.yaml:158]: Given the loss-of-access issues `model-transparency` had with this approach, I would avoid copying the id from `Core Team`

Comment (haydentherapper) [reply github-sync/github-data/sigstore/repositories.yaml]: Done, thanks

File: CODEOWNERS
Comment (haydentherapper) [inline CODEOWNERS:1]: Error will resolve once PR is merged, since community-owners will have write perms added.

**Comment by haydentherapper** [comment]
@cpanato added you as well

**Review by cpanato — APPROVED** [review]

**Review by bobcallaway — APPROVED** [review]

**Review by SantiagoTorres — APPROVED** [review]

**Review by priyawadhwa — APPROVED** [review]

**Pull request merged**
